apps/server/src/infra/courses-client/courses-client.adapter.ts

```diff
@@ -0,0 +1,17 @@
+import { Injectable } from '@nestjs/common';
+import { CourseCommonCartridgeMetadataResponse, CoursesApi, CreateCourseBodyParams } from './generated';
+
+@Injectable()
+export class CoursesClientAdapter {

```

Comment:
why should we move common-cartridge to infra?

Reply:
Because it's a requirement from Arc.